projects/packages/my-jetpack/src/products/class-anti-spam.php

```diff
@@ -106,6 +106,41 @@ public static function get_features() {
 		);
 	}
 
+	/**
+	 * Get the product-slugs of the paid bundles/plans that this product/module is included in
+	 *
+	 * @return array
+	 */
+	public static function get_paid_bundles_that_include_product() {
+		return array(
+			'jetpack_security_t1_yearly',
+			'jetpack_security_t1_monthly',
+			'jetpack_security_t1_bi-yearly',
+			'jetpack_complete',
+			'jetpack_complete_monthly',
+			'jetpack_complete_bi-yearly',
+			'jetpack_business',
+			'jetpack_business_monthly',
+			'jetpack_premium',
+			'jetpack_premium_monthly',
+			'jetpack_personal',
+			'jetpack_personal_monthly',
```

Comment:
Would we like to include WPCOM plans that include Jetpack features or other old Jetpack plans (like Starter) here as well?

Reply:
Considering that WPCOM plans would have a different checkout, etc

Reply:
Yeah, this comment/feedback is no longer relevant. I refactored this so now we're pulling the bundle slugs from the WPCOM features endpoint. I'm marking this comment as "Resolved". 👍